### PR TITLE
feat(player-actions): player actions controller

### PR DESCRIPTION
**Pull request by garrappachc**

### CODE REVIEW

File: src/player-actions-logger/pipes/parse-filters.pipe.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import { PlayerActionEntry } from '../models/player-action-entry';
+
+@Injectable()
+export class ParseFiltersPipe implements PipeTransform {
+  constructor(private readonly playersService: PlayersService) {}
+
+  async transform(input: {
+    [key: string]: string;
+  }): Promise<FilterQuery<PlayerActionEntry>> {
+    if (isEmpty(input)) {
```
Comment (c43721) [inline src/player-actions-logger/pipes/parse-filters.pipe.ts:15]: If input is a string, why pull in a whole lib?

File: e2e/player-action-logs.e2e-spec.ts [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+      const playersService = app.get(PlayersService);
+      const player = await playersService.findBySteamId(players[2]);
+      const authService = app.get(AuthService);
+      authToken = await authService.generateJwtToken(
+        JwtTokenPurpose.auth,
+        player.id,
+      );
+    });
+
+    it('GET /player-action-logs returns 401', async () => {
```
Comment (c43721) [inline e2e/player-action-logs.e2e-spec.ts:48]: All this tests is the auth decorator though?

File: e2e/player-action-logs.e2e-spec.ts [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+      const playersService = app.get(PlayersService);
+      const player = await playersService.findBySteamId(players[0]);
+      const authService = app.get(AuthService);
+      authToken = await authService.generateJwtToken(
+        JwtTokenPurpose.auth,
+        player.id,
+      );
+    });
+
+    it('GET /player-action-logs returns 200', async () => {
```
Comment (c43721) [inline e2e/player-action-logs.e2e-spec.ts:69]: Same here- you just test already tested code though?

**Review by c43721 — CHANGES_REQUESTED** [review]

File: src/player-actions-logger/pipes/parse-filters.pipe.ts (hunk 1 above)
Comment (garrappachc) [reply src/player-actions-logger/pipes/parse-filters.pipe.ts:15]: I just wanted to filter out nulls, undefineds and empty objects... And we're using lodash anyway, so it doesn't really make any difference.

File: e2e/player-action-logs.e2e-spec.ts (hunk 2 above)
Comment (garrappachc) [reply e2e/player-action-logs.e2e-spec.ts:48]: Yes, I want to make sure the decorator (or whatever guards this route) is applied correctly.

File: e2e/player-action-logs.e2e-spec.ts (hunk 3 above)
Comment (garrappachc) [reply e2e/player-action-logs.e2e-spec.ts:69]: It's not covered in unit tests.

Comment (c43721) [reply e2e/player-action-logs.e2e-spec.ts:69]: Weird test then nonetheless

File: e2e/player-action-logs.e2e-spec.ts (hunk 2 above)
Comment (c43721) [reply e2e/player-action-logs.e2e-spec.ts:48]: Weird to test that and not the controller logic. The decorator should have been tested separately

**Pull request merged**